integration_tests/helpers/config.js

```diff
@@ -147,6 +148,11 @@ function createEnv(
     `TARI_MERGE_MINING_PROXY__${network}__PROXY_HOST_ADDRESS`
   ] = `${proxyFullAddress}`;
   configEnvs[`TARI_BASE_NODE__${network}__TRANSPORT`] = "tcp";
+  configEnvs[`TARI_WALLET__${network}__TRANSPORT`] = "tcp";
+  configEnvs[`TARI_WALLET__${network}__TCP_LISTENER_ADDRESS`] =
```

Comment:
Something that is still weird is that I thought these changes would mean we can fix the code above:
```Rust
configEnvs[`TARI_BASE_NODE__${network}__TCP_LISTENER_ADDRESS`] =
    "/ip4/127.0.0.1/tcp/" + (isWallet ? `${walletPort}` : `${baseNodePort}`);
  configEnvs[`TARI_BASE_NODE__${network}__PUBLIC_ADDRESS`] =
"/ip4/127.0.0.1/tcp/" + (isWallet ? `${walletPort}` : `${baseNodePort}`);
```
by removing part that adds the wallet port to that envars but I found when I didn't add the wallet port to those envvars that the wallets couldn't talk to each other so there is still something weird going on.